mm2src/mm2_net/src/wasm/body_stream.rs

```diff
@@ -0,0 +1,399 @@
+use crate::grpc_web::PostGrpcWebErr;
```

Comment:
Could you please write documentation describing this module purpose and provide more info about implementation?

Reply:
done